servicebus/data-plane/azure-servicebus/src/main/java/com/microsoft/azure/servicebus/MessageConverter.java

```diff
@@ -204,18 +169,17 @@ else if (body instanceof AmqpSequence)
                         case ClientConstants.DEADLETTERSOURCENAME:
                             brokeredMessage.setDeadLetterSource((String)entry.getValue());
                             break;
+                        default:
```

Comment:
[SpotBugs-P2] 
This method contains a switch statement where default case is missing. Usually you need to provide a default case.